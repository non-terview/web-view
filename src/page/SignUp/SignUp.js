
import SignupForm from "../../components/SignupForm/SignupForm";


function SignUp() {
    return (
        <SignupForm/>
    );
}

export default SignUp;