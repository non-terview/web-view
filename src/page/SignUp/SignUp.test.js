import SignUp from './SignUp';
import renderer from "react-test-renderer";
import React from "react";


test('SignUp changes the class when hovered', () => {
        const component = renderer.create(
            <SignUp/>,
        );

        let form = component.toJSON();

        expect(form).toMatchSnapshot();
    }
)