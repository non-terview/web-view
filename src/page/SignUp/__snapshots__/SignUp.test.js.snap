// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp changes the class when hovered 1`] = `
<div
  className="sc-gsTEea evOHZW"
>
  <div>
    <div>
      <input
        className="sc-bdfBQB emdnPA"
        name="email"
        onChange={[Function]}
        placeholder="email"
        value=""
      />
    </div>
    <div>
      <input
        className="sc-bdfBQB emdnPA"
        name="password"
        onChange={[Function]}
        placeholder="password"
        value=""
      />
    </div>
    <div>
      <input
        className="sc-bdfBQB emdnPA"
        name="confirmPassword"
        onChange={[Function]}
        placeholder="confirmPassword"
        value=""
      />
    </div>
    <div>
      <input
        className="sc-bdfBQB emdnPA"
        name="name"
        onChange={[Function]}
        placeholder="name"
        value=""
      />
    </div>
  </div>
  <div
    className="sc-dlfnuX logZOV"
    onClick={[Function]}
  >
    회사 가입
  </div>
</div>
`;
